lib/coll.gd

```diff
@@ -2400,6 +2400,38 @@ DeclareOperation( "AsSSortedListNonstored", [IsListOrCollection] );
 DeclareGlobalFunction( "Elements" );
 
 
+#############################################################################
+##
+##  TODO: document FoldLeft
+##
+DeclareGlobalFunction( "FoldLeft" );
+
+
+#############################################################################
+##
+##  TODO: document FoldLeftOp
+##
+DeclareOperation( "FoldLeftOp", [ IsListOrCollection, IsFunction ] );
```

Comment:
TODO: should there also be a declaration for `FoldLeftOp` with three arguments, the third being the initial value?